test/zzzz-bulk_rejection.sh

```diff
@@ -128,7 +128,7 @@ fpod=$( get_running_pod fluentd )
 
 # wait for BulkIndexQueueFull errors in fluentd log
 flog=/var/log/fluentd.log
-os::cmd::try_until_success "grep -q BulkIndexQueueFull /var/log/fluentd.log" $(( 300 * second ))
+os::cmd::try_until_success "grep -q BulkIndexQueueFull /var/log/fluentd.log" $(( 600 * second ))
```

Comment:
it should not take this long - is it possible that fluentd is configured to be a mux client, and is sending logs to mux instead of to es directly?

Reply:
Ahhh, I see...  Thanks, @richm .

That is, if mux is configured, we have to enable DEBUG on mux instead of the client fluentd?  Probably, in that case, the mux's fluentd.log is not visible as /var/log/fluentd.log on the host since it's mounted to the client fluentd not mux?  Let me play with it a little more...

Reply:
@nhosoi For the bulk index test, MUX_CLIENT_MODE should not be set - fluentd should send logs directly to Elasticsearch - mux should not be used.
If this is what is happening, that means there is a problem with some other test - the other test that ran earlier did not unset MUX_CLIENT_MODE from fluentd.
MUX_CLIENT_MODE should be off by default - it is up to each test that tests mux to set MUX_CLIENT_MODE as needed, and reset it back to deleted after the test.

Reply:
Yes, @richm , that was it...  remote-syslog.sh did not clean up the environment variable in the cleanup function.  I pushed the fixed version.